apisix/cli/schema.lua

```diff
@@ -222,7 +222,25 @@ local config_schema = {
                                     }
                                 }
                             }
-                        }
+                        },
+                        key_encrypt_salt = {
+                            anyOf = {
```

Comment:
I know `anyOf` can satisfy the schema check but it should be `oneOf` in this scene?

Reply:
But this is the conventional way in APISIX, it's used everywhere, so i think i should follow this.